packages/angular/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@transifex/angular",
+  "name": "@transifex/angular-projects",
```

Comment:
Here I had this for name: tx-native-angular-sdk, maybe we can use the name of the folder for this?